unit-tests/live/live-common.h

```diff
@@ -65,6 +106,7 @@ inline rs2::device find_first_device_by_name_or_exit( const std::string & dev_na
 
     if( dev_iter != devices_list.end() )
     {
+        test::log.d( "found", repr( *dev_iter ) );
```

Comment:
The enum is not a real char it actually prints the asci code of the char and not the char itself.

See [here](https://rsjenkins.iil.intel.com/job/LRS_libci_pipeline/1792/artifact/win10/win64/static/RelWithDebInfo/unit-tests/test-live-syncer-throughput.log)

-68- found <device: Intel RealSense D455 (S/N: 013322250065  FW: 05.12.15.50  UNLOCKED  on USB3.2)>

Reply:
Weird, when I ran it locally it did put out a character.
Fixed.